src/PKSim.Matlab/MatlabIndividualFactory.cs

```diff
@@ -14,6 +14,7 @@ namespace PKSim.Matlab
    public interface IMatlabIndividualFactory
    {
       ParameterValue[] CreateIndividual(OriginData matlabOriginData, IEnumerable<MoleculeOntogeny> moleculeOntogenies);
```

Comment:
Would that work if you were to pass a parameter as null instead? 
e.g. the first method would be
```
ParameterValue[] CreateIndividual(OriginData matlabOriginData, IEnumerable<MoleculeOntogeny> moleculeOntogenies = null);
```